package.json

```diff
@@ -94,5 +94,5 @@
     "prettier": "^2.7.1",
     "vitest": "^0.34.6"
   },
-  "packageManager": "yarn@1.22.19"
```

Comment:
what will happen if we remove this entry totally?

Reply:
also:
i think will should add this to resolvers entry:
````
  "resolutions": {
    "autoprefixer": "10.4.5",
    "jackspeak": "2.1.1"
  },
````

Reply:
@shootermv would you like to take this PR?